Formula/slides.rb

```diff
@@ -0,0 +1,17 @@
+class Slides < Formula
+  desc "Terminal based presentation tool"
+  homepage "https://github.com/maaslalani/slides"
+  url "https://github.com/maaslalani/slides/archive/refs/tags/v0.1.2.tar.gz"
+  sha256 "195f5269ef78733081fa0034add5dca9c5c855ee01bd990795741cb70e385f23"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args
+  end
+
+  test do
+    assert_match "slides <file.md> [flags]", shell_output("#{bin}/slides")
```

Comment:
We need a test that exercises some actual functionality of the program, and not one that just compares `help` or `version` output. Could you propose a better test here?

Reply:
I may need to change the program since currently it needs a `tty`.

Reply:
We do have ways to simulate a TTY (a PTY) in Homebrew tests. [Something like how these formulae do it](https://github.com/Homebrew/homebrew-core/search?q=pty).

Reply:
I found a similar formula that is very similar to my application (Using bubbletea). Looks like they had similar problems with `PTY` as well.
https://github.com/Homebrew/homebrew-core/blob/11d3369762a7eccf2349eefc732ffe183cb0ba13/Formula/tz.rb

I added a similar test to their formula which should test that the program was at least able to parse the input passed into it.